src/utils/pageProperties.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable consistent-return */
```

Comment:
did splitting this to separate file remove the imports form the integrations? are they slimmer now?

Reply:
Yes